client/lib/domains/resolve-domain-status.js

```diff
@@ -155,8 +163,8 @@ export function resolveDomainStatus(
 			}
 
 			if ( isExpiringSoon( domain, 30 ) ) {
-				const expiresMessage = translate( 'Expires in %(days)s', {
-					args: { days: moment( domain.expiry ).fromNow( true ) },
+				const expiresMessage = translate( 'Domain registration expires on %(expiryDate)s', {
+					args: { expiryDate: moment( domain.expiry ).format( 'LL' ) },
```

Comment:
I think we should leave the message for expiring registrations along for the moment.